src/components/Teams/index.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import Teams from "./Teams";
+import Img from "../../assets/who_we_are_img_1.jpg";
+import "./../../common.css";
+
+export default {
+  component: Teams,
+  title: "Components/Teams",
+};
+
+const teams = [
+  {
+    teamName: "Team test 1",
+    teamComponents: [
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+    ],
+  },
+  {
+    teamName: "Team test 1",
+    teamComponents: [
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+    ],
+  },
+  {
+    teamName: "Team test 1",
+    teamComponents: [
+      {
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+    ],
+  },
+];
+
+export const Primary = () => <Teams teams={teams} />;
+export { default } from "./Teams";
```

Comment:
Shouldn't this file only have this line? It looks like it's a duplicate of `Teams.stories.js`

Reply:
Fixed